tests/api-testing/REST/EditEndpoint.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+const { REST, assert } = require( 'api-testing' );
+const wbk = require( 'wikibase-sdk' )( require( '../wikibase-edit.config' ) );
+const axios = require( 'axios' );
+const requestHelper = require( '../requestHelper' );
+
+describe( 'POST /edit', () => {
+	const basePath = 'rest.php/wikibase-reconcile-edit/v0';
+	const client = new REST( basePath );
+
+	let reconciliationPropertyId, billOfMaterialPropertyId, namePropertyId;
+
+	before( async () => {
+		// needs to maintain a list of what properties map to what "datatype"
+		// could use PropertyLabelResolver to go around knowing the P numbers
+		reconciliationPropertyId = await requestHelper.getOrCreateProperty( 'url', 'identifier' );
+		namePropertyId = await requestHelper.getOrCreateProperty( 'string', 'name' );
+		billOfMaterialPropertyId = await requestHelper.getOrCreateProperty( 'wikibase-item', 'Bill of material' );
```

Comment:
As far as I can tell the common spelling is “bill of material**s**”, not “material”. (Also, I see no reason to write “bill” in uppercase.)

Reply:
done!